site/documentation/entry.js

```diff
@@ -71,19 +71,20 @@ class Entry extends React.Component {
 
     return (
       <div className='border-t border--2 border--gray-faint pt48 pb48 flex-parent flex-parent--wrap'>
-        <div className='txt-s txt-m-mxl col col--12 col--4-ml pr12-ml mb6'>
+        <div className='txt-m col col--12 col--4-ml pr12-ml mb6'>
           <div className='none' id={`expanded-${selectors && selectors[0]}`}>
             {selectorEls}
           </div>
           <div className='inline' id={`collapsed-${selectors && selectors[0]}`}>
             {collapsedSelectorEls}
           </div>
           {expandButton}
-        </div>
-        <div className='col col--12 col--8-ml'>
-          <div className='mb48 prose'>
+
+          <div style={{ fontSize: '13px', lineHeight: '20px' }} className='mt12 mb18 alpha75 txt-s'>
```

Comment:
Inline styling like this defeats the purpose of a css framework. if you aren't happy with our typographic scale, make an issue about it. Not going to merge this.